### DISCUSSION TITLE
select with x-model and x-show not working

### DISCUSSION THREAD

**Discussion by giacomomasseron** [alpinejs/alpine#3270] [1. Help]
This is the code I use:  

	<div class="intro-y box p-5 mt-5" x-data="{ cadenza: 'weekly' }">
		<div class="border border-slate-200/60 dark:border-darkmode-400 rounded-md p-5">
			<select name="cadenza" x:model="cadenza" class="w-full form-control">
				<option value="weekly">Weekly</option>
				<option value="monthly">Monthly</option>
				<option value="annual">Annual</option>
			</select>
			<div x-show="cadenza == 'weekly'" x-transition class="mt-5">
				1
			</div>
			<div x-show="cadenza == 'monthly'" x-transition class="mt-5">
				2
			</div>
			<div x-show="cadenza == 'annual'" x-transition class="mt-5">
				3
			</div>
		</div>
	</div>

The div shown does not change. I always see the first one.

**Comment by SimoTod** [comment]
Typo. Use `x-model` instead of `x:model`

**Reply by giacomomasseron** [reply]
Thank you! sorry for this stupid error
